### DISCUSSION TITLE
Training schema support

### DISCUSSION THREAD

**Discussion by MartinH80** [OpenTracksApp/OpenTracks#1591] [General]
2 months ago I started running again, after not doing any sports for a long time. All surprisingly well so far. Now I want to setup a training schema/plan that includes runs at a specific speed or containing some intervals, e.g. 5x800m with a minute of rest in between. Opentracks allows me to record it, but is there a way to have it tell me to start/stop the intervals? I was thinking about using one of the automation apps, but not sure if it can be done.

**Comment by dennisguse** [comment]
RunnerUp supports this: https://github.com/jonasoreland/runnerup

Adding interval training to OpenTracks would be quite some effort. 
If somebody wants to build it: have fun

**Comment by MartinH80** [comment]
Thanks, I will look into runner-up, although I like opentracks :) Building it myself would be a nice challenge, but I am far from an experienced developer. Who knows. Yesterday I configured a schedule in https://github.com/timer-machine. Although this works on time instead of distance, it might work for me next to opentracks.
